src/common/Tile/index.js

```diff
@@ -40,6 +40,7 @@ const Tile = ({ horizontal, title, year, poster, production, release, descriptio
     <StyledTile horizontal={horizontal}>
       <Poster
         horizontal={horizontal}
+        noPoster={!poster}
```

Comment:
To po zmianach, które później wprowadziłeś nam już się chyba nie przyda ;)

Reply:
Właśnie, że po zmianach to dopiero jest przydatne. Bo bez tego, to zaślepka ma tylko wysokość ikony, wcześniej wszystko miało na sztywno wbite height a teraz trzeba dać tylko tam gdzie nie ma postera.

Reply:
Ok, ale z tego jak śledziłem commity, to ostatecznie wywaliłeś tego propsa ze styled-componentu, więć tutaj już chyba nie będzie potrzebny ;)

Reply:
Dobre, gdzieś na etapie różnego testowania nie zauważyłem, że usunąłem, już przywróciłem bo chociaż działało to brzydko się ikonka wyświetlała

Reply:
Aaa już chyba wiem, o co tutaj chodziło. @TSypko daj znać czy mam rację, po prostu daliśmy object-fit cover dla obrazka, a dla tej zaślepki to kompletnie rozwala widok, robi powiększenie do kamerki, zamiast małej ładnej kamerki?

Reply:
W ogóle teraz wszystko kojarzę :D  Na początku był inny problem, o którym pisałem powyżej, że poster miał wtedy tylko wysokość ikonki, ale to było we wersji gdy był tylko width był na sztywno a wysokość postera sama się ustawiała w zależności od źródła w API. Potem po konsultacji z @katkowa ustaliliśmy, że lepiej będzie jak wszystkie plakaty będą miały tę samą wysokość dlatego ustawiłem z powrotem height na sztywno i ten object fit. Dlatego pewnie uznałem, że ten no poster nie potrzebny ale jest potrzebny, właśnie po to aby ładna ikonka się wyświetlała tak jak teraz wnioskujesz :D